features/promotion/receiving_discount_based_on_nth_order.feature

```diff
@@ -0,0 +1,28 @@
+@promotion
+Feature: Receiving discount based on nth order
+    In order to pay less while placing order
+    As a Customer
+    I want to receive discount for my purchase
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And the store has a product "PHP T-Shirt" priced at "€100.00"
+        And the store ships everywhere for free
+        And the store allows paying offline
+        And there is a promotion "5 order promotion"
+        And it gives "€20.00" off customer's 5th order
+        And I am logged in customer
```

Comment:
:+1: but I think it should be fixed in separate PR (as well as many other minor semantic fixes in steps).